packages/ckeditor5-alignment/src/alignmentediting.js

```diff
@@ -44,24 +45,55 @@ export default class AlignmentEditing extends Plugin {
 		const locale = editor.locale;
 		const schema = editor.model.schema;
 
+		const alignmentOptions = normalizeAlignmentOptions( editor.config.get( 'alignment.options' ) );
+
 		// Filter out unsupported options.
-		const enabledOptions = editor.config.get( 'alignment.options' ).filter( isSupported );
+		const enabledOptions = alignmentOptions.map( option => option.name ).filter( isSupported );
+		const classNameConfig = alignmentOptions.map( option => option.className );
```

Comment:
I'm not sure if we should not stick to the list of objects and transform those to some helping maps.

Reply:
Done.